src/pkg/term/colorizer.go

```diff
@@ -135,3 +136,9 @@ func Fatalf(format string, v ...any) {
 	Errorf("Error: "+format, v...)
 	os.Exit(1)
 }
+
+var ansiRegex = regexp.MustCompile("\x1b(?:\\[[=?]?[0-9;]{0,10}[@-~]|].{0,10}?(?:\x1b\\\\|\x07|$)|[@-Z\\\\^_])")
```

Comment:
Do we have a link for this Regex or explanation for it?

Reply:
yeah let me look for it.